### PR TITLE
Add size and platform options for Satellites

### PR DESCRIPTION
**Pull request by dchw**
Also include a small fix to the OOM hook script, so we dont't have to enable developer mode to make it work.

### CODE REVIEW

**Review by vladaionescu — APPROVED** [review]
🚀

File: release/ami/image.pkr.hcl [hunk 1]
```diff
@@ -19,7 +19,7 @@ locals {
 # https://cloud-images.ubuntu.com/locator/ec2/
 
 source "amazon-ebs" "x86_64" {
-  ami_name      = "earthly-${var.earthly_version}-x86_64-${local.timestamp}"
+  ami_name      = "earthly-${var.earthly_version}-amd64-${local.timestamp}"
```
Comment (dchw) [inline release/ami/image.pkr.hcl:22]: Also a small fix to align the AMI names with the architecture component of our platform strings

Comment (dchw) [reply release/ami/image.pkr.hcl:22]: I already fixed the existing one by copying the AMI to match this naming pattern.

File: cmd/earthly/satellite_cmds.go [hunk 2]
```diff
@@ -7,6 +7,7 @@ import (
 	"os"
 	"time"
 
+	"github.com/jedib0t/go-pretty/table"
```
Comment (alexcb) [inline cmd/earthly/satellite_cmds.go:10]: is there a reason we can't use `"text/tabwriter"` instead?

we already use it in many of our commands, see  https://github.com/earthly/earthly/blob/main/cmd/earthly/account_cmds.go#L442-L443 for an example.

Comment (dchw) [reply cmd/earthly/satellite_cmds.go:10]: It takes a lot more code, but I can rewrite it if you want

Comment (dchw) [reply cmd/earthly/satellite_cmds.go:10]: Rewritten

**Review by alexcb — APPROVED** [review]

File: cmd/earthly/satellite_cmds.go (hunk 2 above)
Comment (alexcb) [reply cmd/earthly/satellite_cmds.go]: thanks! I think it's useful to stay consistent with how we write out these tables.

**Pull request merged**